src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write your code here
+function formatNumber(number) {
+  return number.toLocaleString();
+}
+
+// Отримуємо всі елементи з класом 'population'
+const populations = Array.from(document.querySelectorAll('span.population'));
+
+// Перетворюємо текст у числові значення
+const populationNumbers = populations.map((span) => {
+  const text = span.textContent.trim();
+  const number = parseInt(text.replace(/[^0-9]/g, ''), 10);
+
+  return !isNaN(number) ? number : 0; // Перевірка на число
+});
```

Comment:
Ensure that all elements with the class 'population' contain valid numerical data. If any element contains non-numeric data, it will be converted to 0, which might not be the intended behavior.